Dockerfile.modules

```diff
@@ -0,0 +1,14 @@
+FROM debian:jessie
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+# Install rust toolchain and build mod_prometheus
+ENV PATH=/usr/local/bin:$PATH
+ENV RUST_VERSION=1.15.1
+ENV DOWNLOAD_CHECKSUM=b1e7c818a3cc8b010932f0efc1cf0ede7471958310f808d543b6e32d2ec748e7
+CMD set -ex && apt-get update && apt-get -y install build-essential curl git-core && \
```

Comment:
I'd rather download everything we need at build time and then for `CMD` build the actual module..But I guess it doesn't matter too much.